File: reverse/Reverse.py
from __future__ import print_function
"""
Code by: Keshav Gupta
Edited By: Sanket Chaudhari
"""

def reverse(num_in):
    num_in = int(num); # In case of python3

    if(num_in < 0):
        num_in = (num_in * -1);
        num_in = str(num_in)[::-1];
        num_in = int(num_in) * -1;
        return num_in;
    else:
        num_in = str(num_in)[::-1];
        return int(num_in);

num = input("Enter the number to reverse: ")

print("Reversed Number:",reverse(num));

File: reverse/reverse.go
package main

import (
	"fmt"
	"strconv"
)

func reverse(num int64) int64 {
	var answer, count int64
	for num > 0 {
		mod := num % 10
		answer = 10*answer + mod
		num = num / 10
		count += 1
	}
	return answer
}

func main() {
	fmt.Print("Enter number to reverse: ")
	var input string
	fmt.Scanln(&input)

	num, _ := strconv.ParseInt(input, 10, 64)
	fmt.Printf("Reverse number of %v is %v.\n", input, reverse(num))
}
